### DISCUSSION TITLE
Linking local sibling package in monorepo doesn't work

### DISCUSSION THREAD

**Discussion by JohannesKlauss** [vercel/vercel#5289] [Help]
I have a github repo, that is linked to my vercel account. Inside are several packages, that I want to auto deploy via vercel.
The issue is that the deployment doesn't recognize my local sibling packages and then it fails, because yarn complains, that it cannot find `@project/gql`.

This is my project setup:
```
src
└─ packages
   └─ api (nest.js)
       └─ package.json
   └─ gql (custom library)
       └─ package.json
   └─ page (nextjs)
       └─ package.json
└─ package.json
└─ lerna.json
```

My root package.json
```json
{
  "name": "project",
  "private": true,
  "workspaces": [
    "packages/*"
  ],
  "scripts": {
    "bootstrap": "lerna bootstrap",
  }
}
```

My lerna.json
```
{
  "packages": [
    "packages/*"
  ],
  "useWorkspaces": true,
  "workspaces": ["packages/*"],
  "npmClient": "yarn",
  "version": "independent"
}
```

And inside my nextjs package is the following package.json
```json
{
  "name": "@project/page",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "dev": "next dev -p 8000",
    "build": "next build",
    "start": "next start",
    "storybook": "start-storybook -p 6006 -s ./,./public",
    "build-storybook": "build-storybook"
  },
  "dependencies": {
    // ....
    "@project/gql": "0.1.0", // <------ That is my local sibling package.
    // ....
  },
  "devDependencies": {
    "@babel/core": "7.11.6",
    "babel-loader": "8.1.0",
    "react-is": "16.13.1"
  }
}
```

Locally it works just fine, but I guess vercel doesn't recognize the that it is a local package. I did check the `Include source files outside of the Root Directory in the Build Step.` option.

The root directory for the nextjs app is set to this `packages/page`.

Is there anything I am missing?

**Comment by berndartmueller** [comment]
I have the exact same issue! @JohannesKlauss have you been able to fix it somehow?

**Comment by colebemis** [comment]
I'm having the same issue

**Comment by skene** [comment]
If you delete the build folders of the shared packages locally and try to build the nextjs package, does it still build successfully?

**Comment by ramiel** [comment]
I'm having the same problem. Did you solve it?

**Comment by EMPTYVOID-DEV** [comment]
I'm having the same issue.
